Makefile

```diff
@@ -1,4 +1,4 @@
-OCAMLBUILD=ocamlbuild
+OCAMLBUILD=ocamlbuild -cflag -g
```

Comment:
this should not be necessary because the `true: debug` line in `_tags` should have the same effect. Why did you feel the need for it?